cli/medperf/utils.py

```diff
@@ -245,6 +246,27 @@ def combine_proc_sp_text(proc: spawn) -> str:
     return proc_out
 
 
+class DebugAndOutputLogger(object):
+    """
+    Prints the output both to stderr (directly) and to debug log
+    """
+    def __init__(self):
+        self._just_started_flag = True
+
+    def write(self, byte):
+        if self._just_started_flag:
+            sys.stdout.write("\r\033[K")  # clean the current spinner line
+            self._just_started_flag = False
+        s = byte.decode("utf-8")
+        sys.stdout.write(f'{Fore.WHITE}{Style.DIM}{s}{Style.RESET_ALL}')
+        sys.stdout.flush()
```

Comment:
The `yasmin` spinner interferes too much into the printed lines to ensure statis text is always shown. That's why it's impossible to print dynamically changed logs with escape sequences via `config.ui.write()` and I had to print out everything manually via `sys.stdout`